thoughtful-post/README.md

```diff
@@ -0,0 +1,16 @@
+# Thoughtful-Post-Mailing
+
+## Local development
+
+```
+yarn install
+yarn mailing
+```
+
+## See it Live
+
+https://thoughtful-post-mailing.vercel.app/
+
+## Designs
+
+https://www.figma.com/file/P0vN4dgc5MV8JKMtNWphvN/thoughtful-post-emails
```

Comment:
can you format this like this other README's?

Also maybe a note about local assets here would be useful.